### ISSUE TITLE
Invalid plugin type 'jira' in section 'issues'.

### ISSUE THREAD

**Issue by maxandersen** [psss/did#341]
I've added: 

```
[issues]
type = jira
url = https://issues.redhat.com/
ssl_verify = true
```

and I get:

```
 ERROR  Invalid plugin type 'jira' in section 'issues'.
```

**Comment by maxandersen** [comment]
figured it out - need to do `pip3 install did\[jira\]` for each plugin.

Would be great if the error message would give a hint to that as its not mentioned afaics in the getting started docs :)

**Comment by kquinn1204** [comment]
I encountered the same issue, but I could not get the plugin to successfully install. The failure was associated with```
 ERROR: Failed building wheel for gssapi
Successfully built requests-gssapi
Failed to build gssapi
ERROR: Could not build wheels for gssapi, which is required to install pyproject.toml-based projects
```
I eventually got it installed by (I am running python 3.11.2)

`$ sudo yum install python3.11-devel`

`$ pip install requests-gssapi`

then rerunning `pip3 install did\[jira\]`

**Comment by psss** [comment]
Yes, there are some extra packages needed to be installed on the system for `pip install`:

* https://did.readthedocs.io/en/latest/install.html#pip

Perhaps, we could mention this in the individual plugin documentation as well.
